content/en/docs/instrumentation/js/libraries.md

```diff
@@ -0,0 +1,170 @@
+done like so:
+
+```console
+npm install <name-of-package>
+```
+
+It is typically then registered at application startup time, such as when
+creating a [TracerProvider](/docs/concepts/signals/traces/#tracer-provider).
+
+## Node.js
+
+### Node autoinstrumentation package
+
+OpenTelemetry also defines the
+[auto-instrumentations-node](https://www.npmjs.com/package/@opentelemetry/auto-instrumentations-node)
+metapackage that bundles all Node.js-based instrumentation libraries into a
+single package. It's a convenient way to add automatically-generated telemetry
+for all your libraries with minimal effort.
+
+To use the package, first install it:
+
+```shell
+npm install @opentelemetry/auto-instrumentations-node
+```
+
+Then in your tracing initialization code, use `registerInstrumentations`:
+
+```javascript
+/* tracing.js */
+
+// Require dependencies
+const { getNodeAutoInstrumentations } = require("@opentelemetry/auto-instrumentations-node");
+const opentelemetry = require("@opentelemetry/api");
+const { Resource } = require("@opentelemetry/resources");
+const { SemanticResourceAttributes } = require("@opentelemetry/semantic-conventions");
+const { NodeTracerProvider } = require("@opentelemetry/sdk-trace-node");
+const { registerInstrumentations } = require("@opentelemetry/instrumentation");
+const { ConsoleSpanExporter, BatchSpanProcessor } = require("@opentelemetry/sdk-trace-base");
+
+// This registers all instrumentation packages
+registerInstrumentations({
+  instrumentations: [
+    getNodeAutoInstrumentations()
+  ],
+});
+
+const resource =
+  Resource.default().merge(
+    new Resource({
+      [SemanticResourceAttributes.SERVICE_NAME]: "service-name-here",
+      [SemanticResourceAttributes.SERVICE_VERSION]: "0.1.0",
+    })
+  );
+
+const provider = new NodeTracerProvider({
+    resource: resource,
+});
+const exporter = new ConsoleSpanExporter();
+const processor = new BatchSpanProcessor(exporter);
+provider.addSpanProcessor(processor);
+
+provider.register();
+```
+
+### Using individual instrumentation packages
+
+If you don't wish to use a metapackage, perhaps to decrease your dependency
+graph size, you can install and register individual instrumentation packages.
+
+For example, here's how you can install and register only the
+[instrumentation-express](https://www.npmjs.com/package/@opentelemetry/instrumentation-express)
+and
+[instrumentation-http](https://www.npmjs.com/package/@opentelemetry/instrumentation-http)
+packages to instrument inbound and outbound HTTP traffic.
+
+```shell
+npm install --save @opentelemetry/instrumentation-http @opentelemetry/instrumentation-express
+```
+
+And then register each instrumentation library:
+
+```js
+/* tracing.js */
+
+// Require dependencies
+const { HttpInstrumentation } = require('@opentelemetry/instrumentation-http');
+const { ExpressInstrumentation } = require('@opentelemetry/instrumentation-express');
```

Comment:
Small nit: quote strings consistently. OTel JS uses single-quotes everywhere.

Reply:
So single-string is the standard? I can go through the other docs and update - double quote is used the most in the docs here (and this is just inconsistent with that).